src/libcollections/btree/set.rs

```diff
@@ -509,6 +509,13 @@ impl<T: Ord> Extend<T> for BTreeSet<T> {
     }
 }
 
+#[unstable(feature = "extend_ref", reason = "recently added")]
+impl<'a, T: 'a + Ord + Copy> Extend<&'a T> for BTreeSet<T> {
```

Comment:
Is the `'a` bound required here? The above code for BtreeMap doesn't have `'a` bounds on `K` or `V`, and I'd expect them to have the same requirements.